Makefile

```diff
@@ -30,3 +30,9 @@ bash/fmt:
 
 bash/fmt/check:
 	shfmt -d $(PWD)/rootfs
+
+apk-update:
+	@echo package_repos.md5 old $$(cat package_repos.md5 || echo '<not found>')
+	@docker run --rm -e CLUSTER=galaxy $(DOCKER_IMAGE_NAME) -c \
```

Comment:
galaxy?

Reply:
I don't understand half of what is in the existing Makefile recipes. I just copied `galaxy` from https://github.com/cloudposse/geodesic/blob/05f4ab3001cd561a7b7b1fd5fbd931c9a8cb712f/Makefile#L23

Reply:
Hrmmm I wonder if we still need `CLUSTER`; I think it's an orphaned environment variable.

Reply:
@osterman you added CLUSTER=galaxy. See https://github.com/cloudposse/geodesic/blob/2a97ad046a63db46ec9b34cb7b4559b518c2885a/README.md#quick-start and https://github.com/cloudposse/geodesic/pull/85